tests/testthat/test_plots.R

```diff
@@ -266,6 +266,8 @@ test_that('plot functions are producing desired output',{
   expect_doppelganger_ci("plot.statRes (combined)", plot(imd_anova_res, bw_theme = TRUE))
 
   expect_doppelganger_ci("plot.statRes (combined volcano)", plot(imd_anova_res, plot_type = "volcano", bw_theme = TRUE))
+  
+  expect_doppelganger_ci("plot.statRes (histogram)", plot(imd_anova_res, plot_type = "histogram", fc_colors = c("blue","gray"), bw_theme = TRUE))
```

Comment:
As a note, the historgram forces bw_theme to be true within the function and idk if it makes sense to specify it here as a separate argument